bindings/python/htcondor2/_collector.py

```diff
@@ -28,12 +28,19 @@ def _ad_type_from_daemon_type(daemon_type: DaemonType):
 
 
 class Collector():
+    """
+    FIXME
+    """
 
     #
     # In version 1, there was a distinct DaemonLocation type (a named tuple)
     # that `pool` could also be, but that functionality was never documented.
     #
     def __init__(self, pool = None):
+        """
+
+        :param pool:
+        """
```

Comment:
Should this include a `FIXME` for future documentation?

Reply:
Yes, and I'd actually completely forgotten I hadn't finished the documentation when I asked you to review this PR.  I can do that on this PR if you want more to review, or write up a follow-up ticket.

Reply:
That is up to you, I am fine either way if you want to finish the documentation in this PR or a separate one.

Reply:
This is only urgent enough to keep things more-or-less fresh in my mind, so let's set a good example and do the documentation in the first PR. ;)